django_classified/models.py

```diff
@@ -7,6 +7,10 @@
 from sorl.thumbnail import ImageField
 from unidecode import unidecode
 
+from django.db.models.signals import pre_delete
```

Comment:
@typonaut It should probably be post_delete, so it will only be deleted after the successful deletion of the parent image.